examples/lotr/src/main/python/LOTR-PyRaphtory.ipynb

```diff
@@ -2,53 +2,118 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "source": [
-    "# Lord Of The Rings PyRaphtory Example Notebook 🧝🏻‍♀️🧙🏻‍♂️💍"
-   ],
+   "id": "e233e31e",
    "metadata": {
     "pycharm": {
      "name": "#%% md\n"
     }
-   }
+   },
+   "source": [
+    "# Lord Of The Rings PyRaphtory Example Notebook 🧝🏻‍♀️🧙🏻‍♂️💍"
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "## Setup environment and download data 💾"
-   ],
+   "id": "29bc25aa",
    "metadata": {
     "pycharm": {
      "name": "#%% md\n"
     }
-   }
+   },
+   "source": [
+    "## Setup environment and download data 💾"
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "Import all necessary dependencies needed to build a graph from your data in PyRaphtory. Download csv data from github into your tmp folder (file path: /tmp/lotr.csv)."
-   ],
+   "id": "8da8d0d0",
    "metadata": {
     "pycharm": {
      "name": "#%% md\n"
     }
-   }
+   },
+   "source": [
+    "Import all necessary dependencies needed to build a graph from your data in PyRaphtory. Download csv data from github into your tmp folder (file path: /tmp/lotr.csv)."
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": 1,
-   "source": [
-    "#pip install pyvis matplotlib"
-   ],
-   "outputs": [],
+   "id": "ba406737",
    "metadata": {
     "pycharm": {
      "name": "#%%\n"
     }
-   }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: pyvis in /Users/lucasjeub/opt/miniconda3/lib/python3.10/site-packages (0.3.0)\n",
```

Comment:
Might want to clear this just as its got your name in it.

Reply:
done